core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java

```diff
@@ -136,11 +136,23 @@ private SharedServiceConfigsView createSharedServiceConfigView(Stack stack) {
             sharedServiceConfigsView.setRangerAdminPort(DEFAULT_RANGER_PORT);
             sharedServiceConfigsView.setDatalakeCluster(false);
             sharedServiceConfigsView.setAttachedCluster(true);
+        } else if (blueprintUtils.isSharedServiceReadyBlueprint(blueprint)) {
```

Comment:
qq should we just check the type of the stack ?

Reply:
+1

Reply:
should. but this is how it was, I can rewrite if you insist

Reply:
we need 3 branch here 
WORKLOAD  -> this is an attached cluster
DATALAKE -> this is a datalake
LEGACY ->  not attached to any datalake or environment